appsettings.json

```diff
@@ -13,6 +13,10 @@
   "LocalConfig": {
     "UseLocalDb": "false"
   },
+  "EmailConfig": {
+    "Email": "gitfood.fun@gmail.com",
+    "Password": ""
```

Comment:
mamy email bez hasła xD?

Reply:
I wsm hasła nie możemy prztrzymywać na gitcie bo to troche będzie security issue

Reply:
Dlatego jest puste, uzupełnie to na serwerze

Reply:
gitówa :)